### PR TITLE
Add a reference to PHP 8.2

### PR DESCRIPTION
**Pull request by mikelittle**

### CODE REVIEW

File: docs/specifications.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ order: 100
 
 The following versions are available, and the specific version in use will depend on your configuration.
 
-- PHP Version: 7.4, 8.0, 8.1
+- PHP Version: 7.4, 8.0, 8.1, 8.2 See [Altis compatibility chart](docs://guides/updating-php/#altis-compatibility-chart)
```
Comment (owaincuvelier) [inline docs/specifications.md:10]: I think we should add an "experimental" tag to PHP 8.2 for now. Thought?

Comment (mikelittle) [reply docs/specifications.md:10]: The linked compatibility chart notes that both 8.1 and 8.2 are experimental.

File: docs/specifications.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ order: 100
...
 - MySQL Version: 5.6, 5.7 (compatible; using AWS Aurora)
```
Comment (ferschubert-hm) [inline docs/specifications.md:11]: Can we remove 5.6 version is no longer in use and has been fully deprecated also by AWS Aurora.

**Review by ferschubert-hm — CHANGES_REQUESTED** [review]
Left a comment to remove deprecated MySQL version as well

File: docs/specifications.md (hunk 2 above)
Comment (mikelittle) [reply docs/specifications.md]: Done. Thanks

**Review by ferschubert-hm — APPROVED** [review]

**Pull request merged**